javascripts/controllers/LoginCtrl.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+app.controller("LoginCtrl", function($location, $rootScope, $scope, AuthService){
+	$scope.authenticate = () => {
+		AuthService.authenticateGoogle().then((result) => {
```

Comment:
Not that this was part of this PR technically, but because you have it in here, make sure to allow this auth method on your firebase dashboard.

Reply:
Thanks.  Done.